### DISCUSSION TITLE
Text Column Length Cap?

### DISCUSSION THREAD

**Discussion by t673606** [supabase/supabase#21988] [Questions]
Hi, I am trying to store som html code inside og a text column in my table, I have read the size limit is 1 gb, but i am no where near that limit with my amount of text. I start of fine, but for some reason it is cut short after line 30 or so. I set the column via an API call in buildship now, but have tried setting it from inside flutterflow as well, but i got the same problem then.

Does anyone know what I might do about this? The API call works fine

![Skjermbilde 2024-03-13 kl  09 14 42](https://github.com/supabase/supabase/assets/116776820/e2719d14-51bb-4275-ac6b-8427790bc5dd)

**Comment by GaryAustin1** [comment]
Does the data read back correctly from the API or SQL editor?
If so then probably just a limit on the dashboard UI.

**Reply by t673606** [reply]
Yes, I reads back properly. But the full text are not visible on the front-end either

**Reply by GaryAustin1** [reply]
I tested a large chunk of text and could get it all back in the SQL editor. The dashboard table UI did not show all of it until I hit the expand box and got the data.  I also inserted the data (200k) in the UI.
What do you mean by front-end?  
![image](https://github.com/supabase/supabase/assets/54564956/9fe4d6d0-48f9-473b-9ab4-c62f60a6980e)
![image](https://github.com/supabase/supabase/assets/54564956/d5902e77-a28a-454c-ad3e-0bb69aef5e55)

**Reply by t673606** [reply]
This does not seam to be the problem, maybe its a limit to how much i can upload at once?
